src/main/java/in/projecteka/datanotificationsubscription/UserServiceProperties.java

```diff
@@ -0,0 +1,31 @@
+package in.projecteka.datanotificationsubscription;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Getter;
+import org.springframework.boot.context.properties.ConfigurationProperties;
+import org.springframework.boot.context.properties.ConstructorBinding;
+
+@ConfigurationProperties(prefix = "subscriptionmanager.userservice")
+@Getter
+@AllArgsConstructor
+@ConstructorBinding
+@Builder
+public class UserServiceProperties {
+    private final String url;
```

Comment:
I think we should be able to get rid of most of the properties here. We will not use anything related to otp_attempt, health_id_search_attempt, enable_share.

Reply:
Done